.github/CONTRIBUTING.md

```diff
@@ -16,6 +16,7 @@ Note
 - If you plan to add some new features that involve large changes, it is encouraged to open an issue for discussion first.
 - If you are the author of some papers and would like to include your method to mmdetection,
```

Comment:
- If you are the author of some papers and would like to include your method to mmdetection,
please let us know (open an issue or contact the maintainers). We will much appreciate your contribution.